### DISCUSSION TITLE
堆叠面积图支持渐变色

### DISCUSSION THREAD

**Discussion by yuruwang** [antvis/G2Plot#2613] [Ideas]
目前单个的面积图可以通过areaStyle配置fill实现颜色渐变，但是对于堆叠面积图（还有堆叠柱状图等等...）并不能在areaStyle的callback参数中获取当前面积渲染的color值。需求想实现的效果如下：
![image](https://user-images.githubusercontent.com/31492974/120985572-5210c600-c7ae-11eb-964d-b8496f2d4968.png)
能不能把j渐变色的模式作为内置的功能支持一下？或者在areaStyle的callback中增加当前渲染的color色值？业务对这个需求的要求比较高

**Comment by webkubor** [comment]
阿里这个项目感觉支持不够高，都没有人维护
